test/e2e/resize_nodes.go

```diff
@@ -124,6 +126,38 @@ func createServiceWithNameSelector(c *client.Client, ns, name string) error {
 	return err
 }
 
+func newPodOnNode(podName, nodeName string, image string) *api.Pod {
```

Comment:
This function only creates an API object. It is created in a given namespace in createServeHostnamePodOnNode